### DISCUSSION TITLE
blue background on folder icon

### DISCUSSION THREAD

**Discussion by haratosan** [nvim-tree/nvim-tree.lua#1603] [Q&A]
![image](https://user-images.githubusercontent.com/55960058/191972201-31e9f5dd-dea0-4122-8926-f8b8fa3b3498.png)

hi there
since i updated all the packages i get these blue squares behind the folder-icons.
does anyone know how to remove them?

thx
fabian

**Comment by alex-courtis** [comment]
Which packages did you upgrade? vim or system?

What do you see in the `File Icons` section of `:help nvim-tree-introduction`: "... should look like an open folder" ?

Do you have any `NvimTree` highlight groups defined?

**Comment by haratosan** [comment]
I upgrades the vim-packages with :PackerUpdate and had to change a few settings after that.
The folder icon looks fine:
![image](https://user-images.githubusercontent.com/55960058/192084367-dbd29132-54e0-4b27-b24d-0bb77882d881.png)

my NvimTree setup looks like this:
```
require'nvim-tree'.setup {
  renderer = {
    highlight_git = true,
    highlight_opened_files = 'all',
    root_folder_modifier = ':~',
    icons = {
      padding = ' ',
      show = {
        file = true,
        folder = true,
        folder_arrow = false,
      },
    },
  },

  actions = {
    open_file = {
      resize_window = true,
    },
  },

  disable_netrw       = true,
  hijack_netrw        = true,
  open_on_setup       = true,
  ignore_ft_on_setup  = {},
  open_on_tab         = true,
  hijack_cursor       = true,
  update_cwd          = false,

  update_focused_file = {
    enable      = false,
    update_cwd  = false,
    ignore_list = {}
  },

  system_open = {
    cmd  = nil,
    args = {}
  },

  view = {
    width = 30,
    side = 'left',
    mappings = {
      custom_only = false,
      list = {}
    }
  }
}

```

**Reply by alex-courtis** [reply]
It _looks_ like your patched font is OK; the background is just in the tree itself.

It looks like you have no highlight groups defined.

Actions:
1. test with an [nvt-min.lua](https://github.com/kyazdani42/nvim-tree.lua/issues/new?assignees=&labels=bug&template=bug_report.yml) as per a bug report
2. run `:so /usr/share/nvim/runtime/syntax/hitest.vim` and examine `NvimTree` groups
